src/main/java/com/projetogerenciamentocurso/gerenciamentocurso/controller/AlunoController.java

```diff
@@ -0,0 +1,62 @@
+package com.projetogerenciamentocurso.gerenciamentocurso.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.projetogerenciamentocurso.gerenciamentocurso.dto.AlunoDto;
+import com.projetogerenciamentocurso.gerenciamentocurso.models.Aluno;
+import com.projetogerenciamentocurso.gerenciamentocurso.service.AlunoService;
+
+@RestController
+@RequestMapping(path = AlunoController.PATH)
+public class AlunoController {
+
+	public static final String PATH = "/aluno";
+
+	@Autowired
+	private AlunoService alunoService;
+
+	@PostMapping
+	public ResponseEntity<Aluno> salvarAluno(@RequestBody AlunoDto aluno) {

```

Comment:
Sugiro criar um novo DTO só para a resposta, pois o negócio pode pedir pra retornar somente alguns campos
```suggestion
	public ResponseEntity<AlunoRespostaDto> salvarAluno(@RequestBody AlunoDto aluno) {
```

Reply:
feito